_posts/en/2020-12-04-better-tech-through-micro-procurementpt2.md

```diff
@@ -0,0 +1,52 @@
+date: "2020-12-04"
+---
+<!--markdownlint-disable MD033-->
+This is part two of a two-part series looking at micro-procurement for digital work.
+In this second post, we look at what can make implementing micro-procurement challenging, as well as a sneak peek at what we are planning at ESDC.
+
+***
+
+## What’s stopping the GC from doing micro-procurement of digital work
+
+The web of procurement rules is real.
+Procurement officers with extensive training help us navigate that web.
+They help ensure that things are done by the book and no-one loses their financial signing authority or worse, gets fired.
+Like all policy, though, procurement policy is open to interpretation and because of this, appetite for risk (by that procurement officer, procurement team, CFO or Deputy Head) can be a factor in implementing any procurement innovation.
+It can be seen as less risky to do things the way they have always been done.
+Related, a "fear of audit" was noted in the [Internal Red Tape Reduction report](https://internal-red-tape-reduction-report.github.io/chapter-3/#root-causes) as a driver of "the need to document every step of the process, which, in turn, drives the heavy paperwork, all in paper format."
+But as the Office of the Procurement Ombudsman pointed out in their most recent [annual report](http://opo-boa.gc.ca/rapports-reports/2019-2020/index-eng.html), simplifying and streamlining procurement in the GC "may require a higher degree of risk tolerance.”
+
+There is another challenge related to procurement policy.
+We know that flexibility exists within GC procurement policy to establish a simplified process for LDV procurements.
+However this flexibility exists because policy doesn't explicitly say it cannot be done.
+As a result, there is very little guidance on how to do micro-procurement within the policy documents or at the departmental level.
+The Office of the Procurement Ombudsman noted this in their [2019 Low Dollar Value Report](http://opo-boa.gc.ca/faiblevaleur-dollarvalue-eng.html) stating that "The audit found that despite adopting the Government of Canada’s contracting policies, TBS's internal process was not supported by a sufficiently articulated policy framework specific to LDV contracting "particularly for non-expert contracting authorities."
+
+Departmental procurement rules can also create barriers to establishing micro-procurement initiatives in the GC.
+Unlike policy, departmental procurement rules can vary widely from department to department.
+This is especially true of rules around credit card use.
+Policy doesn't dictate things like credit card limits and individual transaction limits but departmental rules do.
+When looking into establishing a micro-procurement initiative in the GC, a good first step would be to learn your department's rules around credit card use.
+
+On the IT side, in order for micro-procurement of digital work to be possible, we must break our work out into small chunks.
```

Comment:
We're saying "digital work". I wonder if you can expand a bit on that while while not confuse the reader or increasing the length of the blog too much. Maybe add a couple of sentences at the end of the paragraph, or even a new paragraph all together:

Digital Work is software, which constitutes two main parts: the source code (text that can be read and re-used by developers), and the object code (the compiled version of the source code running on your computer).
This opens up new opportunities for Digital Work collaboration as we, should we chose to, could share openly the source code of our software to others, and have others do the same. Large IT initiatives gets broken down into smaller pieces of these source code bits to work on, instead of large (and often highly complex) endeavours.

this is where @gcharest OSS presentation stuff could be linked or content re-used 👍 

If not... subject for another blog? :)

> We will be talking about what Digital Work constitutes and the opportunities it represents in an upcoming blog...

Reply:
This is a really good point. I've adjusted that paragraph to include the concepts you mention above @remyb2canada. Let me know what you think! If I haven't captured it, please let me know! :)